QBTracker/QubitSDK/Utils/QBDevice.swift

```diff
@@ -30,7 +31,7 @@ class QBDevice {
     
     private static func randomStringInMd5() -> String {
         let timestamp = NSDate().timeIntervalSince1970
-        let randomValue = Int(timestamp) * Int(arc4random_uniform(10))
+        let randomValue = Int(timestamp) * Int.random(in: 1...10)
```

Comment:
Nice! Should we explicitly use Int64? This would make the code safer on 32 bit platforms (I don't know if any users are on 32 bit platform tbh, [unlikely](https://blakespot.com/ios_device_specifications_grid.html))

Also increase the upper bound for random to 1000 to reduce collision for users accessing the app for the first time and at the same exact time. 

```swift
let randomValue = Int64(timestamp) * Int64.random(in: 1...1000)
```

Reply:
Since SDK support the minimum iOS version of 11.0 which dropped the support of the 32-bit then we shouldn't add explicit use of Int64 cause by default Int is used as a type alias for Int64.